### ISSUE TITLE
Weekly review freeze

### ISSUE THREAD

**Issue by drnola** [brandonkboswell/weekly-review#5]
Love the plugin. Used it quite frequently in the past with no problem. Just fired it up and obsidian freezes. I restarted, uninstalled etc.. but it still doesn't work. Please let me know if there' something I can do. Thank you

**Comment by brandonkboswell** [comment]
So each time you launch the plugin Obsidian freezes?

**Comment by drnola** [comment]
Yes. I repeated several time and got same result. I had to force restart.

**Comment by brandonkboswell** [comment]
If you go into the Plugin preferences and set the "How many days to show?" to 1 does it not crash?

![BKB 2023-09-19 at 07 26 09](https://github.com/brandonkboswell/weekly-review/assets/921925/42d98631-c5e4-4842-81e6-53bdcab68c4b)

**Comment by drnola** [comment]
I’ll try that and let you know

**Comment by drnola** [comment]
That solved the problem. I went up to 6 days. And then I went to 7 and it froze. I restarted and went back in and the space for number of days was blank. I put in the number 1 and it worked but when I returned to the plugin, it was blank. I repeated a few times and each time it was blank. I uninstalled and then reinstalled and changed base number to 3 and it is working again. I'm not going to put it past 3.
